modules/custom/d_commerce/modules/d_commerce_products_list/templates/views-view-unformatted--droopler_commerce_products_list.html.twig

```diff
@@ -15,9 +15,9 @@
  * @see template_preprocess_views_view_unformatted()
  */
 #}
-<div class="row">
+<div class="row mx-n3 mb-3">
```

Comment:
a co to w sumie jest `-n3` ?

Reply:
ujemny margines ;-) klasa row ma defaultowo ujemny na grid-gutter-width/2, a tu mamy zmniejszony padding dla elementow `col`, wiec trzeba ustawic odpowiedni ujemny dla `row`